astropy/io/fits/connect.py

```diff
@@ -302,3 +306,200 @@ def write_table_fits(input, output, overwrite=False):
 io_registry.register_reader('fits', Table, read_table_fits)
 io_registry.register_writer('fits', Table, write_table_fits)
 io_registry.register_identifier('fits', Table, is_fits)
+
+
+def read_nddata_fits(filename, ext_data=0, ext_meta=0, ext_mask='mask',
+                     ext_uncert='uncert', kw_unit='bunit', dtype=None,
+                     **kwargs_for_open):
+    """
+    Read data from a FITS file and wrap the contents in a \
+    `~astropy.nddata.NDData`.
+
+    Parameters
+    ----------
+    filename : str and other types
+        see :func:`~astropy.io.fits.open` what possible types are allowed.
+
+    ext_data, ext_meta, ext_mask, ext_uncert : str or int, optional
+        Extensions from which to read ``data``, ``meta``, ``mask`` and
+        ``uncertainty``.
+        Default is ``0`` (data), ``0`` (meta), ``'mask'`` (mask) and
+        ``'uncert'`` (uncertainty).
```

Comment:
I feel like a default of ``uncert`` is too specialized for a base reader. Also, I'm confused as to why ``meta`` is mapped to an extension not to keywords? (I'll need to review more to find out!)